app-typescript/components/Toast.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import * as ReactDOM from "react-dom";
+import Text from './Text';
+import { IOptions } from './Wrapper';
+import Wrapper from './Wrapper';
+
+const TOAST_ID = "react-toast";
+
+class Toasted {
+    createToast?(message: string | React.ReactNode, options: IOptions): void;
+
+    constructor() {
+        let element;
+        const existingElement = document.getElementById(TOAST_ID);
+
+        if (existingElement) {
+            element = existingElement;
+        } else {
+            const el = document.createElement("div");
+            el.id = TOAST_ID;
+            el.className = "sd-toast__container sd-toast__container--top ng-scope";
+            if (document.body != null) {
+                document.body.appendChild(el);
+            }
+            element = el;
+        }
+        ReactDOM.render(
+            <Wrapper notify={this.bind as any} />, element);
```

Comment:
avoid type casting using `as`. In this case, you can use a ref and you won't need neither `this.bind` nor `notify` prop for `Wrapper`